Add note about the IDEA problem (#282)
vaadin/docs: articles/guide/contributing/editor-settings-intellij-idea.asciidoc

the License.
```

> == Known problems in configuration

Reviewer: ```suggestion
== Known Problems In Configuration
```

IntelliJ IDEA has a known problem with link:https://youtrack.jetbrains.com/issue/IDEA-192220[importing sources of other module as resources].
Which is the case with the Flow project where `flow-client` has `flow-server` sources as a resource.

The fix for this is to during development time comment out the `flow-server` resource for the `flow-client/pom.xml`.

> .Commment out the flow-server resource

Reviewer: ```suggestion
.Comment out the flow-server resource
```

```
<resource><directory>../flow-server/src/main/java</directory></resource>
```

[NOTE]

> To build the `flow-client` package remember to uncomment the resource.

Reviewer: ```suggestion
To build the `flow-client` package remember to un-comment the resource.
```

> [NOTE]

Reviewer: ```suggestion
.Build flow-client package
[NOTE]
```
